app/helpers/green_lanes_helper.rb

```diff
@@ -8,16 +8,16 @@ def exemption_checkbox_none?(category_assessment_id)
   end
 
   def render_exemptions_or_no_card(category, assessments, result)
-    no_exemptions = assessments.public_send("no_cat#{category}_exemptions")
-    exemptions_met = assessments.public_send("cat_#{category}_exemptions_met")
-    total_exemptions = assessments.public_send("cat_#{category}_exemptions")
+    no_exemptions = assessments.send("no_cat#{category}_exemptions")
+    assessments_met = assessments.send("cat_#{category}_assessments_met")
+    total_assessments = assessments.send("cat_#{category}_assessments").pluck(:category_assessment_id).map(&:to_s)
 
-    all_exemptions_met = total_exemptions.count == exemptions_met.count
+    all_assessments_met = total_assessments.count == assessments_met.count
```

Comment:
This needed to change to count the assessments rather than the exemptions as some assessments have multiple exemptions where only one exemption needs to count